README.md

```diff
@@ -32,9 +32,15 @@ The `sns-testing` solution is based on Docker; however, there are subtle issues
      ```bash
      brew install coreutils jq
      ```
-   Also make sure you have Rust installed including the `wasm32-unknown-unknown` target
-   that you can add into your active toolchain by running:
 
+   You also need rosetta that you can install by running:
+   ```bash
+   softwareupdate --install-rosetta
```

Comment:
I think I never needed to run this command  — where does it come from? (I thought we don’t rely on emulation/ binary translation anymore).